package.json

```diff
@@ -3,7 +3,6 @@
   "description": "A backend for the Atocha app",
   "version": "1.0.0",
   "main": "\"./server/index.js\"",
-  "type": "module",
```

Comment:
So you were able to get it working without ES6 terminology (e.g. import/export?)

Reply:
Yeah as long as you use "require", you shouldn't need it as far as I know